src/main/resources/DDL.sql

```diff
@@ -0,0 +1,30 @@
+CREATE DATABASE my_ticket_service
+    WITH
+    OWNER = postgres
+    ENCODING = 'UTF8'
+    LOCALE_PROVIDER = 'libc'
+    CONNECTION LIMIT = -1
+    IS_TEMPLATE = False;
+
+
+CREATE TABLE public."User"
```

Comment:
1. do not use `public` in the name of table
2. `user` is reserved keyword in dbs. use for example: 
```sql
create table user_info(...)
```
instead of `User`
Note: good practice to use names if tables in lower case + "_" sign

Reply:
I took names of tables as they were written in the task. Yes, I had problem with reserved name "User", but I just put extra quotes \" in SQL queries. I will change it.

Reply:
quotes is bad practice, because if you use them in schema, you will be forced to use them in other queries